index.html

```diff
@@ -236,14 +236,15 @@ <h2 class="contact-form-title">
             coding. I'd love to help with it
         </p>
         <form action="https://formspree.io/f/xknarzyz" class="contact-form" method="post">
-            <input maxlenth="30" type="text" class="input" value="Phone Khaing Hein" name="name" placeholder="Enter Name"  required>
-            <input type="email" class="input" value="pkh2662003@gmail.com" name="email" placeholder="Enter Email"  required>
-            <textarea maxlenth="30" name="description" class="textarea" cols="30" rows="7"
-                placeholder="Write your message here"  required></textarea>
-            <button type="button" class="contact-form-button">Get in touch</button>
+            <input maxlenth="30" type="text" class="input" value="Phone Khaing Hein" name="name"
+                placeholder="Enter Name" required>
+            <input type="email" class="input" value="pkh2662003@gmail.com" name="email" placeholder="Enter Email"
+                required>
+            <textarea maxlenth="500" name="message" class="textarea" cols="30" rows="7"
+                placeholder="Write your message here" required></textarea>
+            <button type="submit" class="contact-form-button">Get in touch</button>
         </form>
     </section>
```

Comment:
- [ ] As stated by the previous reviewer, I would encourage you to style the form with the use of a flexbox so that it would be a part of you and also lay the tags appropriately.
![image](https://user-images.githubusercontent.com/86200356/220583357-ca079b2a-09f9-4df7-b738-59728c7e6f9a.png)

Reply:
Fixed.